src/utils/sources.ts

```diff
@@ -86,14 +91,13 @@ export const getSourcePayload = (values: any, testSources: SourceWithRepository[
 
     if (!isTestSourceExists) {
       notificationCall('failed', 'Provided test source does not exist');
-      return;
     }
```

Comment:
Why without early return instead?

Reply:
Don't like it, I prefer using `if..else` in such cases. This hung `return` makes me feel bad tbh :)